packages/monitoring/src/serviceMonitor/alivenessChecker.ts

```diff
@@ -0,0 +1,56 @@
+import { events } from "../helpers/events";
+import { IServiceAliveness } from "../types";
+import { disconnectServices } from "./disconnectHandler";
+
+/**
+ * Monitors the aliveness of one or multiple services at a specified interval.
+ *
+ * @param services - An array of services implementing the IServiceAliveness interface to be monitored.
+ * @param interval - The interval, in minutes, at which the services' aliveness will be checked. Defaults to 2 minutes.
+ *
+ * @throws Will throw an error if the provided services array is empty.
+ *
+ * @returns An object with a method `exitAndDisconnect` that can be used to stop the monitoring and disconnect services.
+ *
+ */
+export function monitorServiceAliveness(
+  services: IServiceAliveness[],
+  interval = 2,
+): { exitAndDisconnect: () => Promise<void> } {
+  if (!services.length) throw new Error("No services to monitor");
```

Comment:
be careful with your checks, do services.length === 0, while your code works, still better to be as explicit as possible